nexus-repository-helm/src/main/java/org/sonatype/repository/helm/internal/AssetKind.java

```diff
@@ -25,18 +25,35 @@
  */
 public enum AssetKind
 {
-  HELM_INDEX(CacheControllerHolder.METADATA),
-  HELM_PROVENANCE(CacheControllerHolder.CONTENT),
-  HELM_PACKAGE(CacheControllerHolder.CONTENT);
+  HELM_INDEX(CacheControllerHolder.METADATA, ".yaml"),
+  HELM_PROVENANCE(CacheControllerHolder.CONTENT, ".tgz.prov"),
+  HELM_PACKAGE(CacheControllerHolder.CONTENT, ".tgz");
 
   private final CacheType cacheType;
+  private final String extension;
 
-  AssetKind(final CacheType cacheType) {
+  AssetKind(final CacheType cacheType, final String extension) {
```

Comment:
Question: May it happen that helm asset kind has more than one extension type? Maybe it's better to make a `Set` for extensions?

Reply:
Discussed